lib/core/utils.js

```diff
@@ -448,18 +448,8 @@ RESUtils.currentSubreddit = function(check) {
 	}
 };
 RESUtils.thingsContainer = function(body) {
-	var container;
 	body = body || document.body;
-	container = body.querySelectorAll(RESUtils.thing.prototype.containerSelector);
-	// stupid sponsored links create a second div with ID of sitetable (bad reddit! you should never have 2 IDs with the same name! naughty, naughty reddit!)
```

Comment:
yay cleaning out cruft  (reddit changed it to `sitetable_organic` after whatever release added this in)